stl/inc/xutility

```diff
@@ -3246,10 +3249,38 @@ namespace ranges {
         }
     };
 
-    // VARIABLE ranges::distance
     inline constexpr _Distance_fn distance{_Not_quite_object::_Construct_tag{}};
 
-    // CLASS ranges::_Next_fn
+    // VARIABLE ranges::ssize
+    class _Ssize_fn { // Per the proposed resolution of LWG-3403
+    public:
+        // clang-format off
+        template <class _Rng>
+        _NODISCARD constexpr auto operator()(_Rng&& _Range) const requires requires { _RANGES size(_Range); } {
+            using _SSSsssigned = _Make_signed_like_t<decltype(_RANGES size(_Range))>;
+            using _Ty          = typename _Parsel_type<is_integral_v<_SSSsssigned>>::template _Apply<_SSSsssigned>;

```

Comment:
Heh. :snake: I think this is probably a bit too silly for production code, though :joy_cat:

Reply:
You're always ruining Casey's fun! :(

Reply:
I started with `_Sty` (for "signed type") and thought "A 'sty' is where pigs live - this needs more snek!" We really need to up our memecode game ;)